### PR TITLE
[KYUUBI #1105][FOLLOWUP] Minor improve `ZorderBytesUtils.defaultValue`

### PR DESCRIPTION
**Pull request by cfmcgrady**
### _Why are the changes needed?_

fix [comment](https://github.com/apache/incubator-kyuubi/pull/1109#discussion_r711007709)

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by cfmcgrady** [comment]
cc @ulysses-you @yaooqinn

**Review by pan3793 — APPROVED** [review]

File: dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/ZorderBytesUtils.scala [hunk 1]
```diff
@@ -171,30 +171,33 @@ object ZorderBytesUtils {
...
+        Int.MaxValue
+      case LongType =>
+        Long.MaxValue
+      case FloatType =>
+        Float.MaxValue
+      case DoubleType =>
+        Double.MaxValue
+      case StringType =>
+        UTF8String.fromBytes(new Array[Byte](0))
+      case TimestampType =>
```
Comment (yaooqinn) [inline dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/ZorderBytesUtils.scala:192]: to L184

File: dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/ZorderBytesUtils.scala [hunk 2]
```diff
@@ -171,30 +171,33 @@ object ZorderBytesUtils {
...
+        Long.MaxValue
+      case DateType =>
+        Int.MaxValue
+      case _: DecimalType =>
```
Comment (yaooqinn) [inline dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/ZorderBytesUtils.scala:196]: ditto

File: dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/ZorderBytesUtils.scala [hunk 3]
```diff
@@ -171,30 +171,33 @@ object ZorderBytesUtils {
...
+        Long.MaxValue
+      case FloatType =>
+        Float.MaxValue
+      case DoubleType =>
+        Double.MaxValue
+      case StringType =>
+        UTF8String.fromBytes(new Array[Byte](0))
+      case TimestampType =>
+        Long.MaxValue
+      case DateType =>
```
Comment (yaooqinn) [inline dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/ZorderBytesUtils.scala:194]: to L182

**Review by ulysses-you — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master for 1.4.0

**Pull request closed without merging**